Source/Rows/Common/FieldRow.swift

```diff
@@ -269,28 +288,23 @@ open class _FieldCell<T> : Cell<T>, UITextFieldDelegate, TextFieldCell where T:
             if let titleLabel = titleLabel, let text = titleLabel.text, !text.isEmpty {
                 views["label"] = titleLabel
                 dynamicConstraints += NSLayoutConstraint.constraints(withVisualFormat: "H:[imageView]-(15)-[label]-[textField]-|", options: NSLayoutFormatOptions(), metrics: nil, views: views)
-                dynamicConstraints.append(NSLayoutConstraint(item: textField,
-                                                             attribute: .width,
-                                                             relatedBy: (row as? FieldRowConformance)?.textFieldPercentage != nil ? .equal : .greaterThanOrEqual,
-                                                             toItem: contentView,
-                                                             attribute: .width,
-                                                             multiplier: (row as? FieldRowConformance)?.textFieldPercentage ?? 0.3,
-                                                             constant: 0.0))
-            } else {
+				if let calculatedTitlePercentage = calculatedTitlePercentage {
+					dynamicConstraints.append(NSLayoutConstraint(item: titleLabel, attribute: .width, relatedBy: .equal, toItem: contentView, attribute: .width, multiplier: calculatedTitlePercentage, constant: 0.0))
+				}
```

Comment:
As described [here](https://github.com/xmartlabs/Eureka/pull/965#issuecomment-318293995), I would go for this

```
if let calculatedTitlePercentage = calculatedTitlePercentage {
	dynamicConstraints.append(NSLayoutConstraint(item: titleLabel,
	                                             attribute: .width,
	                                             relatedBy: (row as? FieldRowConformance)?.titlePercentage != nil ? .equal : .lessThanOrEqual,
	                                             toItem: contentView,
	                                             attribute: .width,
	                                             multiplier: calculatedTitlePercentage,
	                                             constant: 0.0))
}
```